test/test_quantized_models.py

```diff
@@ -50,11 +50,18 @@ def check_script(self, model, name):
     def _test_classification_model(self, name, input_shape):
         # First check if quantize=True provides models that can run with input data
 
-        model = torchvision.models.quantization.__dict__[name](pretrained=False, quantize=True)
+        if name in ['inception_v3', 'googlenet']:
+            model = torchvision.models.quantization.__dict__[name](pretrained=False, quantize=True, init_weights=True)
+        else:
+            model = torchvision.models.quantization.__dict__[name](pretrained=False, quantize=True)
```

Comment:
I believe those changes in the test are only needed because we currently raise an error, but if we instead send a warning this wouldn't be necessary, right?

Reply:
Yeah. For warning it's not required.

Reply:
If it's not required, maybe it might be worth removing this then, except if the tests are now much faster to run?

Reply:
That's not required. Removed.